-- 2024-01-15 00:15:32.2399 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 00:15:32.4141 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 00:17:56.7962 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 00:17:56.8331 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 00:21:34.3872 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 00:21:34.4305 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 00:22:38.7393 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 00:22:38.7756 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:15:01.3733 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:15:01.4704 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:18:03.5380 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:18:03.5818 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:29:55.3247 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:29:55.3646 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:33:47.3920 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:33:47.4304 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:34:17.9128 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:34:17.9545 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:35:37.1386 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:35:37.1747 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:36:16.2727 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:36:16.3176 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:36:59.3369 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:36:59.4055 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:38:55.7869 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:38:55.8259 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:52:05.2938 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:52:05.3389 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:53:10.8767 ERROR[74] | ICHI_CORE.Controllers.MasterController.AuthController.Register. 
      MSG_CONTENT:System.InvalidOperationException: Unable to track an instance of type 'UserRole' because it does not have a primary key. Only entity types with a primary key may be tracked.
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.GetOrCreateEntry(Object entity, IEntityType entityType)
   at Microsoft.EntityFrameworkCore.Internal.InternalDbSet`1.EntryWithoutDetectChanges(TEntity entity)
   at Microsoft.EntityFrameworkCore.Internal.InternalDbSet`1.AddAsync(TEntity entity, CancellationToken cancellationToken)
   at ICHI_CORE.Controllers.MasterController.AuthController.Register(UserRegister userRegister) in D:\Code\ICHI_API\Controllers\MasterController\AuthController.cs:line 74
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:53:27.0531 ERROR[74] | ICHI_CORE.Controllers.MasterController.AuthController.Register. 
      MSG_CONTENT:System.InvalidOperationException: Unable to track an instance of type 'UserRole' because it does not have a primary key. Only entity types with a primary key may be tracked.
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.GetOrCreateEntry(Object entity, IEntityType entityType)
   at Microsoft.EntityFrameworkCore.Internal.InternalDbSet`1.EntryWithoutDetectChanges(TEntity entity)
   at Microsoft.EntityFrameworkCore.Internal.InternalDbSet`1.AddAsync(TEntity entity, CancellationToken cancellationToken)
   at ICHI_CORE.Controllers.MasterController.AuthController.Register(UserRegister userRegister) in D:\Code\ICHI_API\Controllers\MasterController\AuthController.cs:line 74
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:57:01.0153 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:57:01.0590 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:58:05.0776 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:58:05.1140 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:58:17.2997 ERROR[75] | ICHI_CORE.Controllers.MasterController.AuthController.Register. 
      MSG_CONTENT:Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Invalid column name 'data_id'.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__209_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:fc322814-932d-4952-ac5c-3eb5e0f2b324
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at ICHI_CORE.Controllers.MasterController.AuthController.Register(UserRegister userRegister) in D:\Code\ICHI_API\Controllers\MasterController\AuthController.cs:line 75
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:58:48.5724 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:58:48.6163 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 16:58:53.8959 ERROR[75] | ICHI_CORE.Controllers.MasterController.AuthController.Register. 
      MSG_CONTENT:Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Invalid column name 'data_id'.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__209_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:209d6270-7e98-499c-83c2-1ee0207bb225
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at ICHI_CORE.Controllers.MasterController.AuthController.Register(UserRegister userRegister) in D:\Code\ICHI_API\Controllers\MasterController\AuthController.cs:line 75
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:00:22.2019 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:00:22.2624 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:02:12.4297 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:02:12.4657 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:02:33.3199 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:02:33.3600 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:04:40.0095 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:04:40.0478 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:06:49.2525 ERROR[75] | ICHI_CORE.Controllers.MasterController.AuthController.Register. 
      MSG_CONTENT:Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Violation of PRIMARY KEY constraint 'PK_user_role'. Cannot insert duplicate key in object 'dbo.user_role'. The duplicate key value is (2).
The statement has been terminated.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__209_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:77068dc3-4894-4620-b1ca-d8730cd16665
Error Number:2627,State:1,Class:14
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at ICHI_CORE.Controllers.MasterController.AuthController.Register(UserRegister userRegister) in D:\Code\ICHI_API\Controllers\MasterController\AuthController.cs:line 75
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:06:56.4677 ERROR[75] | ICHI_CORE.Controllers.MasterController.AuthController.Register. 
      MSG_CONTENT:Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Violation of PRIMARY KEY constraint 'PK_user_role'. Cannot insert duplicate key in object 'dbo.user_role'. The duplicate key value is (2).
The statement has been terminated.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__209_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:77068dc3-4894-4620-b1ca-d8730cd16665
Error Number:2627,State:1,Class:14
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at ICHI_CORE.Controllers.MasterController.AuthController.Register(UserRegister userRegister) in D:\Code\ICHI_API\Controllers\MasterController\AuthController.cs:line 75
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:07:01.1170 ERROR[75] | ICHI_CORE.Controllers.MasterController.AuthController.Register. 
      MSG_CONTENT:Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Violation of PRIMARY KEY constraint 'PK_user_role'. Cannot insert duplicate key in object 'dbo.user_role'. The duplicate key value is (2).
The statement has been terminated.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__209_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:77068dc3-4894-4620-b1ca-d8730cd16665
Error Number:2627,State:1,Class:14
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at ICHI_CORE.Controllers.MasterController.AuthController.Register(UserRegister userRegister) in D:\Code\ICHI_API\Controllers\MasterController\AuthController.cs:line 75
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:09:42.9218 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:09:42.9500 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:09:42.9708 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:10:21.2078 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:10:21.2356 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:10:21.2574 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:14:22.3371 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:14:22.3665 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:14:22.3850 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:15:38.0456 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:15:38.0725 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:15:38.0909 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:43:20.3738 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:43:20.4080 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:43:20.4291 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:43:35.9650 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:43:35.9884 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:43:36.0103 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:45:15.3278 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:45:15.3556 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:45:15.3741 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:45:29.5724 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:45:29.6095 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:45:29.6302 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:46:33.4640 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:46:33.4916 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:46:33.5100 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:46:38.0915 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:46:38.1227 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:46:38.1433 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:47:08.3738 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:47:08.4275 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:47:38.7150 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:47:38.7619 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:49:11.4505 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:49:11.4901 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:50:01.7623 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:50:01.7977 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:54:05.4488 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:54:05.4760 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:54:05.4956 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:54:11.1416 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:54:11.1701 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 17:54:11.1888 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:06:58.5866 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:06:58.6143 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:06:58.6331 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:11:38.0851 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:11:38.1160 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:11:38.1377 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:11:47.0283 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:11:47.0567 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:11:47.0755 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:12:18.7382 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:12:18.7664 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:12:18.7854 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:12:22.9908 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:12:23.0181 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:12:23.0383 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:13:23.8890 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:13:23.9167 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:13:23.9357 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:13:28.8940 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:13:28.9247 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:13:28.9477 ERROR[49] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 49
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:32:34.0008 INFO[38] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:32:34.0398 ERROR[39] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:33:06.2993 ERROR[63] | ICHI_CORE.Controllers.MasterController.AuthController.Register. 
      MSG_CONTENT:Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The INSERT statement conflicted with the FOREIGN KEY constraint "FK_user_role_role_role_id". The conflict occurred in database "demo", table "dbo.role", column 'Id'.
   at Microsoft.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at Microsoft.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at Microsoft.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at Microsoft.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at Microsoft.Data.SqlClient.SqlDataReader.TryHasMoreRows(Boolean& moreRows)
   at Microsoft.Data.SqlClient.SqlDataReader.TryReadInternal(Boolean setTimeout, Boolean& more)
   at Microsoft.Data.SqlClient.SqlDataReader.ReadAsyncExecute(Task task, Object state)
   at Microsoft.Data.SqlClient.SqlDataReader.InvokeAsyncCall[T](SqlDataReaderBaseAsyncCallContext`1 context)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.ConsumeResultSetAsync(Int32 startCommandIndex, RelationalDataReader reader, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.ConsumeAsync(RelationalDataReader reader, CancellationToken cancellationToken)
ClientConnectionId:6e53b659-0e51-431c-9526-f59d4c49de2b
Error Number:547,State:0,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.ConsumeAsync(RelationalDataReader reader, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at ICHI_CORE.Controllers.MasterController.AuthController.Register(UserRegister userRegister) in D:\Code\ICHI_API\Controllers\MasterController\AuthController.cs:line 63
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 18:34:15.7886 ERROR[63] | ICHI_CORE.Controllers.MasterController.AuthController.Register. 
      MSG_CONTENT:Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): The INSERT statement conflicted with the FOREIGN KEY constraint "FK_user_role_role_role_id". The conflict occurred in database "demo", table "dbo.role", column 'Id'.
   at Microsoft.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at Microsoft.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at Microsoft.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at Microsoft.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at Microsoft.Data.SqlClient.SqlDataReader.TryHasMoreRows(Boolean& moreRows)
   at Microsoft.Data.SqlClient.SqlDataReader.TryReadInternal(Boolean setTimeout, Boolean& more)
   at Microsoft.Data.SqlClient.SqlDataReader.ReadAsyncExecute(Task task, Object state)
   at Microsoft.Data.SqlClient.SqlDataReader.InvokeAsyncCall[T](SqlDataReaderBaseAsyncCallContext`1 context)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.ConsumeResultSetAsync(Int32 startCommandIndex, RelationalDataReader reader, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.ConsumeAsync(RelationalDataReader reader, CancellationToken cancellationToken)
ClientConnectionId:6e53b659-0e51-431c-9526-f59d4c49de2b
Error Number:547,State:0,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.AffectedCountModificationCommandBatch.ConsumeAsync(RelationalDataReader reader, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at ICHI_CORE.Controllers.MasterController.AuthController.Register(UserRegister userRegister) in D:\Code\ICHI_API\Controllers\MasterController\AuthController.cs:line 63
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 21:42:36.8616 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 21:42:37.1648 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 21:42:37.7420 ERROR[69] | Program.<Main>$. 
      MSG_CONTENT:System.InvalidOperationException: No service for type 'ICHI_CORE.Entities.DbInitializer.IDbInitializer' has been registered.
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService[T](IServiceProvider provider)
   at Program.<<Main>$>g__SeedDatabase|0_3(<>c__DisplayClass0_0&) in D:\Code\ICHI_API\Program.cs:line 83
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 69
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 21:44:45.1067 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 21:44:45.1511 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 21:45:04.3126 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 21:45:04.3494 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 21:46:31.6478 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 21:46:31.6877 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 21:47:19.4781 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 21:47:19.5134 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 21:57:01.8413 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 21:57:01.8760 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:01:31.0253 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:01:31.0766 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:11:16.3628 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:11:16.4000 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:14:18.7894 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:14:18.8267 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:21:01.1060 ERROR[54] | ICHI_CORE.Controllers.BaseController.BaseController`1.FindAll. 
      MSG_CONTENT:Product | FindAll | Exception: System.Data.SqlTypes.SqlNullValueException: Data is Null. This method or property cannot be called on Null values.
   at Microsoft.Data.SqlClient.SqlDataReader.GetFieldValueFromSqlBufferInternal[T](SqlBuffer data, _SqlMetaData metaData, Boolean isAsync)
   at Microsoft.Data.SqlClient.SqlDataReader.GetFieldValueInternal[T](Int32 i, Boolean isAsync)
   at Microsoft.Data.SqlClient.SqlDataReader.GetFieldValue[T](Int32 i)
   at lambda_method130(Closure, QueryContext, DbDataReader, ResultContext, SingleQueryResultCoordinator)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at API.Helpers.PagedList`1.CreateAsync(IQueryable`1 source, Int32 pageNumber, Int32 pageSize)
   at ICHI_CORE.Controllers.BaseController.BaseController`1.FindAll(PaginationParams paginationParams) in D:\Code\ICHI_API\Controllers\BaseController\BaseController.cs:line 54
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:21:28.5195 ERROR[54] | ICHI_CORE.Controllers.BaseController.BaseController`1.FindAll. 
      MSG_CONTENT:Product | FindAll | Exception: System.Data.SqlTypes.SqlNullValueException: Data is Null. This method or property cannot be called on Null values.
   at Microsoft.Data.SqlClient.SqlDataReader.GetFieldValueFromSqlBufferInternal[T](SqlBuffer data, _SqlMetaData metaData, Boolean isAsync)
   at Microsoft.Data.SqlClient.SqlDataReader.GetFieldValueInternal[T](Int32 i, Boolean isAsync)
   at Microsoft.Data.SqlClient.SqlDataReader.GetFieldValue[T](Int32 i)
   at lambda_method130(Closure, QueryContext, DbDataReader, ResultContext, SingleQueryResultCoordinator)
   at Microsoft.EntityFrameworkCore.Query.Internal.SingleQueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at API.Helpers.PagedList`1.CreateAsync(IQueryable`1 source, Int32 pageNumber, Int32 pageSize)
   at ICHI_CORE.Controllers.BaseController.BaseController`1.FindAll(PaginationParams paginationParams) in D:\Code\ICHI_API\Controllers\BaseController\BaseController.cs:line 54
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:22:06.7184 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:22:06.7672 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:29:21.8501 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:29:21.8851 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:29:37.7237 INFO[240] | ICHI_CORE.Controllers.BaseController.BaseController`1.CreateBatch. 
      MSG_CONTENT:CategoryProduct | CreateBatch | TotalRecord: 1
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:34:04.9895 INFO[240] | ICHI_CORE.Controllers.BaseController.BaseController`1.CreateBatch. 
      MSG_CONTENT:CategoryProduct | CreateBatch | TotalRecord: 10
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:39:14.5785 INFO[240] | ICHI_CORE.Controllers.BaseController.BaseController`1.CreateBatch. 
      MSG_CONTENT:CategoryProduct | CreateBatch | TotalRecord: 10
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:39:54.2632 INFO[240] | ICHI_CORE.Controllers.BaseController.BaseController`1.CreateBatch. 
      MSG_CONTENT:CategoryProduct | CreateBatch | TotalRecord: 11
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:49:26.9515 INFO[240] | ICHI_CORE.Controllers.BaseController.BaseController`1.CreateBatch. 
      MSG_CONTENT:CategoryProduct | CreateBatch | TotalRecord: 11
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:55:20.1624 INFO[240] | ICHI_CORE.Controllers.BaseController.BaseController`1.CreateBatch. 
      MSG_CONTENT:CategoryProduct | CreateBatch | TotalRecord: 5
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 22:57:25.5044 INFO[240] | ICHI_CORE.Controllers.BaseController.BaseController`1.CreateBatch. 
      MSG_CONTENT:CategoryProduct | CreateBatch | TotalRecord: 5
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:06:44.3732 INFO[240] | ICHI_CORE.Controllers.BaseController.BaseController`1.CreateBatch. 
      MSG_CONTENT:CategoryProduct | CreateBatch | TotalRecord: 5
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:09:35.0679 INFO[240] | ICHI_CORE.Controllers.BaseController.BaseController`1.CreateBatch. 
      MSG_CONTENT:CategoryProduct | CreateBatch | TotalRecord: 5
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:11:41.5087 INFO[240] | ICHI_CORE.Controllers.BaseController.BaseController`1.CreateBatch. 
      MSG_CONTENT:CategoryProduct | CreateBatch | TotalRecord: 5
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:13:06.4263 INFO[240] | ICHI_CORE.Controllers.BaseController.BaseController`1.CreateBatch. 
      MSG_CONTENT:CategoryProduct | CreateBatch | TotalRecord: 5
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:16:44.3945 INFO[240] | ICHI_CORE.Controllers.BaseController.BaseController`1.CreateBatch. 
      MSG_CONTENT:CategoryProduct | CreateBatch | TotalRecord: 5
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:20:21.0217 INFO[240] | ICHI_CORE.Controllers.BaseController.BaseController`1.CreateBatch. 
      MSG_CONTENT:CategoryProduct | CreateBatch | TotalRecord: 5
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:23:03.5862 INFO[240] | ICHI_CORE.Controllers.BaseController.BaseController`1.CreateBatch. 
      MSG_CONTENT:CategoryProduct | CreateBatch | TotalRecord: 5
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:40:00.7902 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:40:00.8272 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:40:00.8535 ERROR[50] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 50
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:40:08.2659 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:40:08.2940 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:40:08.3141 ERROR[50] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 50
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:40:38.3681 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:40:38.3958 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:40:38.4150 ERROR[50] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 50
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:40:42.6145 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:40:42.6422 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:40:42.6610 ERROR[50] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 50
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:41:18.8453 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:41:18.8680 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:41:18.8871 ERROR[50] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 50
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:43:22.8713 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:43:22.8945 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:43:22.9137 ERROR[50] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 50
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:48:03.6669 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:48:03.6946 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:48:03.7135 ERROR[50] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 50
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:48:08.0984 INFO[39] | Program.<Main>$. 
      MSG_CONTENT:Get data from appSettings success.
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:48:08.1262 ERROR[40] | Program.<Main>$. 
      MSG_CONTENT:Test log error
      EXXCEPTION:  | MESSAGE:
-- 2024-01-15 23:48:08.1462 ERROR[50] | Program.<Main>$. 
      MSG_CONTENT:Microsoft.Extensions.Hosting.HostAbortedException: The host was aborted.
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.ThrowHostAborted()
   at Microsoft.Extensions.Hosting.HostFactoryResolver.HostingListener.OnNext(KeyValuePair`2 value)
   at System.Diagnostics.DiagnosticListener.Write(String name, Object value)
   at Microsoft.Extensions.Hosting.HostBuilder.ResolveHost(IServiceProvider serviceProvider, DiagnosticListener diagnosticListener)
   at Microsoft.Extensions.Hosting.HostApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.WebApplicationBuilder.Build()
   at Program.<Main>$(String[] args) in D:\Code\ICHI_API\Program.cs:line 50
      EXXCEPTION:  | MESSAGE:
